### PR TITLE
Update 2.0.10

### PR DESCRIPTION
**Pull request by ryanskeith**
Log:
- Update Version and Hash
- Updated build sripts
- Change requirements based off of input from last builder

Notes:
- This change enables Python 3.11

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,17 +16,19 @@ requirements:
...
     - llvmdev
     - llvm-tools
     - libclang
     - maturin <=0.12.0
     - toml
-    - pip
     - winpty
   host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Just for curiosity, why does it require compilers in `host`?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   name: pywinpty
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pywinpty/pywinpty-{{ version }}.tar.gz
-  sha256: 20ec117183f79642eff555ce0dd1823f942618d65813fb6122d14b6e34b5d05a
+  sha256: cdbb5694cf8c7242c2ecfaca35c545d31fa5d5814c3d67a4e628f803f680ebea
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:12]: Missing skipping `py<37`, see https://github.com/andfoy/pywinpty/blob/2869fbf311b3d4320fe07f3882f1bafdd54b1f0c/pyproject.toml#L3

File: recipe/meta.yaml [hunk 3]
```diff
@@ -16,17 +16,19 @@ requirements:
...
-    - pip
     - winpty
   host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - setuptools
+    - wheel
     - python
     - pip
     - winpty
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:32]: Can you add `pip check` too?

**Review by ELundby45 — APPROVED** [review]
LGTM. `license_family` could be added.

**Review by varlackc — COMMENTED** [review]
as soon as the license family is added we should be able to approve

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**